jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/FragmentConfiguration.java

```diff
@@ -24,6 +24,9 @@
  */
 public class FragmentConfiguration extends AbstractConfiguration
 {
+    // Holds a Map<Resource, Resource> .
+    //  key: Resource to the Jar
+    //  value: Resource to the web fragment xml
     public static final String FRAGMENT_RESOURCES = "org.eclipse.jetty.webFragments";
```

Comment:
This constant should now have a `ee10` specifier, since there is an equivalent class for `ee9`.
I would use the package name, eg. `org.eclipse.jetty.ee10.webapp.webFragments`.